<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreeATE Results</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExport"/>
   <addaction name="actionFirstPage"/>
   <addaction name="actionPrevPage"/>
   <addaction name="actionNextPage"/>
   <addaction name="actionLastPage"/>
   <addaction name="actionSearch"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_testcase">
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>68</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>TestCase</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_testcase">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="tableView_case">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_detial">
   <property name="windowTitle">
    <string>Details</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_detial">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="tableView_detail">
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(232, 232, 232);</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/search.png</normaloff>:/search.png</iconset>
   </property>
   <property name="text">
    <string>search</string>
   </property>
   <property name="toolTip">
    <string>search results with barcode</string>
   </property>
  </action>
  <action name="actionPrevPage">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ChevronLeft.png</normaloff>:/ChevronLeft.png</iconset>
   </property>
   <property name="text">
    <string>PrevPage</string>
   </property>
   <property name="toolTip">
    <string>Previous Page</string>
   </property>
  </action>
  <action name="actionFirstPage">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/FristPage.png</normaloff>:/FristPage.png</iconset>
   </property>
   <property name="text">
    <string>FirstPage</string>
   </property>
   <property name="toolTip">
    <string>First Page</string>
   </property>
  </action>
  <action name="actionNextPage">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ChevronRight.png</normaloff>:/ChevronRight.png</iconset>
   </property>
   <property name="text">
    <string>NextPage</string>
   </property>
   <property name="toolTip">
    <string>Next page</string>
   </property>
  </action>
  <action name="actionLastPage">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/LastPage.png</normaloff>:/LastPage.png</iconset>
   </property>
   <property name="text">
    <string>LastPage</string>
   </property>
   <property name="toolTip">
    <string>Last Page</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/output.png</normaloff>:/output.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export results</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
